ingress-nginx/values.yaml

```diff
@@ -1,8 +1,16 @@
 ingress-nginx:
   controller:
+    kind: "DaemonSet"
```

Comment:
I don't think we want a daemonset: if we e.g. scaled to 50 nodes for some random job then, we don't want an ingress on every node.